README.md

```diff
@@ -38,43 +41,51 @@ Hyperion enables itself automatically for all Activities in your application. To
 
 To embed the Hyperion Plugin Menu into your own Activity, acquire a `PluginViewFactory` from `Hyperion#getPluginViewFactory` and call `create(Activity)`:
 
-```
+```Java
```

Comment:
I normally see these all lowercase but maybe it doesn't matter?

Reply:
I was using some vscode extension from my react native setup, but it seemed to like this casing